src/twisted/test/test_udp.py

```diff
@@ -577,13 +635,25 @@ class MulticastTests(TestCase):
     if not interfaces.IReactorMulticast(reactor, None):
         skip = "This reactor does not support multicast"
 
+    interface: str = "0.0.0.0"
```

Comment:
Maybe just a quick note about these values, if someone will check them in the future and ask why these are class members.

```suggestion
    # We define a bunch of test parameters to allow sharing
    # code between IPv4 and IPv6 tests.
    interface: str = "0.0.0.0"
```